src/entities/userLists.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * A single user list, with fields based on the data found in
+ * `src/references/default-token-lists.json`.
+ */
+export interface UserList {
```

Comment:
we have a pattern of exporting to `entities/index` (you can check out the other ones that are already in there) and only importing from `entities` instead of having to do `entities/userList` etc!

Reply:
Great, thanks! I pushed an update that should address this and your second comment. I'll also add the `handlers/web3` updates when I'm done with them later today.